lib/dco.js

```diff
@@ -27,14 +27,17 @@ module.exports = function (commits) {
         signedOff = false
         defaults.failure.description = `The sign-off is missing.`
       } else {
-        if (!validator.validate(commit.author.email)) {
+        let email = commit.committer ? commit.committer.email : commit.author.email
+        let name = commit.committer ? commit.committer.name : commit.author.name
```

Comment:
Now that I'm seeing this, I wonder if it should attempt to check against both committer and author. 

For example, I could open a pull request with:

1. commits from me, in which case it could match either `committer` or `author`
2. commits from someone else that already included a sign-off, in which case it would match the `author`
2. commits from someone that didn't include a sign-off but I added a sign-off, in which case it would match the `committer`

Reply:
The issue I ran into that prompted this PR was the case where I had signed a commit with one account (committer and email are the same) and later rebased and signed the commit with a different account (only committer). In this case, the author and committer will be different which was unexpected because, in my case, there was only a single commit on a PR and I could visually see it was signed by email address b (but the DCO check was looking for address a)

Reply:
I agree checking both committer and author seems like the more robust solution. And, really, the above situation I got in is likely one that could be solved outside of this tool, perhaps by updating the README with more details on how this works and common troubleshooting steps.